apps/files/js/app.js

```diff
@@ -116,6 +118,7 @@
 					maxChunkSize: OC.appConfig.files && OC.appConfig.files.max_chunk_size
 				}
 			);
+			this.currentFileList = this.fileList
```

Comment:
```suggestion
			this.updateCurrentFileList(this.fileList);
```

Does the same, but seems cleaner if we have the explicit setter :)